scripts/run-tests.sh

```diff
@@ -13,7 +14,12 @@ then
     sh -e /etc/init.d/xvfb start &
     sleep 3
 fi
-bundle exec rake rubocop features:smoke
+if [[ "$SEXY_SETTINGS" =~ .*poltergeist.* ]]
+then
+    bundle exec rake rubocop features:smoke_except
```

Comment:
`smoke` tests are clear test group definition. Unfortunately, smoke_except is not. Moreover it introduces one more group which is not documented and will be replaced with next Howitzer upgrade procedure. What about marking some scenarios which we are not willing to be executed under Poltergeist driver with `no_poltergeist` tag. I would exclude all scenarios when marked with the tag for **ALL** group of tests when current driver is `poltergeist`